### DISCUSSION TITLE
Dynamically Build Booked dates

### DISCUSSION THREAD

**Discussion by nathanmedhurstschoolshr** [easepick/easepick#58] [General]
Hi!

Another question from me, this time with regards to booking out individual dates.

This is what i have so far:
https://jsfiddle.net/4L9zctjr/1/

My setup will be as follows:
User Selects dropdown on front end
This triggers back end database call
values are stored in a hidden input formatted like this:
[['2022-04-01','2022-04-08'],['2022-05-09','2022-05-13'],['2022-05-02','2022-05-06'],['2022-09-05','2022-09-09']]
(This is from a live database call)
I grab the value of the hidden input to be used in the code - however this does not work as shown in the jsfiddle example

However if i replace it with just the text, it works "MOSTLY" fine

The other issue i have is for the second section from the 9th May until the 13th May it still allows me to select the 13th May which i dont think it should.

Anyone able to assist?

**Comment by nathanmedhurstschoolshr** [comment]
So ive managed to figure this out, the problem was when you type it manually it see's it as an array, when you grab the value from an input it doesnt, so you need to convert the string to an array.

**Comment by crossiono** [comment]
do you have an example code for this?

**Reply by nathanmedhurstschoolshr** [reply]
This is what i did: https://jsfiddle.net/7bo08afL/

Hopefully it helps, you have to PARSE the JSON and add some replace regex

**Reply by crossiono** [reply]
thanks
